recipe/meta.yaml

```
@@ -1,34 +1,34 @@
 {% set name = "trio" %}
-{% set version = "0.26.2" %}
+{% set version = "0.30.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/trio-{{ version }}.tar.gz
-  sha256: 0346c3852c15e5c7d40ea15972c4805689ef2cb8b5206f794c9c19450119f3a4
+  sha256: 0781c857c0c81f8f51e0089929a26b5bb63d57f927728a5586f7e36171f064df
 
 build:
-  skip: true  # [py<38]
+  skip: true  # [py<39]
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - pip
-    - setuptools >=64
+    - setuptools >=77
     - wheel
   run:
     - python
     - attrs >=23.2.0
-    - cffi >=1.14  # [win]
+    - cffi >=1.14
```

```suggestion
    - cffi >=1.14   # [win]
```
The `os_name == 'nt'` part in their [pyproject.toml](https://github.com/python-trio/trio/blob/v0.30.0/pyproject.toml#L51) will exclude this dependency unless on Windows.